test/src/server/helpers/collections.js

```diff
@@ -29,8 +29,10 @@ import orm from '../../../bookbrainz-data';
 
 
 const {expect} = chai;
+/* eslint-disable import/no-commonjs */
 chai.use(require('chai-sorted'));
 chai.use(require('chai-arrays'));
```

Comment:
These can be imported with ES6 module syntax instead (see [this example](https://github.com/bookbrainz/bookbrainz-site/blob/939b54fa9233ffc9c3da45f5e56ba8029361e1cb/test/src/server/routes/collection.js#L5))  to resolve the eslint issue.
Same goes for `test/src/server/helpers/revisions.js`

Reply:
Done!